.env

```diff
@@ -15,3 +15,9 @@ SENTRY_DNS=
 
 # Log level to use, DEBUG by default in dev
 LOG_LEVEL=DEBUG
+
+POSTGRES_HOST=postgres
+POSTGRES_DB=postgres
+POSTGRES_USER=postgres
+POSTGRES_PASSWORD=postgres
+POSTGRES_EXPOSE=127.0.0.1:5432
```

Comment:
see also direnv for local settings, it's handy (see [how it can be used in product opener](https://openfoodfacts.github.io/openfoodfacts-server/dev/how-to-use-direnv/))